### ISSUE TITLE
[EN][Quest][Hellfire Peninsula] A Spirit Guide

### ISSUE THREAD

**Issue by Atlntssplayer** [Atlantiss/BugTracker#8600]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

This quest had problems in the past and have been reported here and the ticket was closed with comment "fixed".
Well now it has problems again.

**Description**: The quest can't be completed because you can't talk to the NPC that should finish the quest for you.

**Current behaviour**: When you start the quest, you use a quest item at the appropriate place which summons an 'ancestral spirit wolf' NPC which you have to follow.
This NPC wolf should give the player a buff called "Guided by the Spirits" which would make player being able to talk to the NPC called 'Gorkan Bloodfist' to finish the quest.
This buff would give the player enough reputation with The Mag'har faction which would let the player be able to speak to said NPC.
The spirit wolf NPC gains the buff but the player character doesn't and when you arrive to the NPC which could complete the quest there is no question mark above it's head and every NPC is unfriendly from that faction and you can't talk to the neccessary NPC.

**Expected behaviour**: The spirit wolf should give the player the buff which would allow the player to speak to the NPC that would finish the quest.

**Steps to reproduce the problem**:

1. Do the pre-req. quests in Thrallmar
2.  Pick up this quest chain and finish quests until you are on the "A Spirit Guide" quest.
3.  Go to the place where you have to use the quest item
4. Use the quest item and summon the spirit wolf NPC then take notice if it gives you the "Guided by the Spirits" buff or not.
5. Follow the NPC until it stops at NPC called 'Gorkan Bloodfist'
6. See if you can complete the quest here or not.

**Revision**: 5258

How the quest should work can be seen in this video:
https://www.youtube.com/watch?v=xFF8DOSeMcg

Also there is a lot of info about how it should work here:
http://www.wowhead.com/quest=9410/a-spirit-guide#comments:id=1622004

thx.

**Comment by tomatosz** [comment]
Currently, a (rather long) workaround is to get Mag'har rep by killing ogres in Nagrand. Once you're no longer hostile, you can finish the quest.

**Comment by Electro1293** [comment]
Yeah, still bugged, normally spirit wolf should finish to talk and quest will complete, but here doesn't work.

**Comment by attan0** [comment]
bump

**Comment by VgqGHTNm** [comment]
Shameless bump to confirm this issue still exists as of today.

**Comment by narkil** [comment]
Bump.
Could you please fix this one ? Killing almost 300 ogres is a huge waste of time, and it shouldnt be so hard to fix this one.

**Comment by AbigailKnox** [comment]
Quest is still bugged. :(

**Comment by Supernova335** [comment]
Bump. 
It's not a fix, but a work around that doesn't include killing 300 ogers. Go to Jorin Deadeye in Garadar in Nagrand (he is friendly no matter your rep with Mag'har) at 55, 37 and pick up the quest "The Impotent Leader". Upon turning in that quest to Kilrath at 19, 60 in Terokkar Forest you will recieve the rep needed to become Neutral with the Mag'har. I really hope this helps.

**Comment by godfuckyou** [comment]
Bump.
Found a better way to by-pass this quest. If you go to Nagrand and kill Ogers until you get neutral rep with the Mag'har and then you can complete the Spirit Guide quest without the buff.

**Comment by godfuckyou** [comment]
a

**Comment by blueninja1800** [comment]
Bump.
Found a by-pass to complete this quest. If you go to Nagrand and kill Ogers until you get neutral rep with the Mag'har, you can go Gorkan Bloodfist and complete the quest without the buff.

**Comment by ameleyanigi** [comment]
Bump.
